pages/common/git-effort.md

```diff
@@ -0,0 +1,29 @@
+# git effort
+
+> Display how much activity a file has had, showing commits per file and "active days" i.e. total number of days that contributed to the file.
+> Part of `git-extras`.
+> More information: <https://github.com/tj/git-extras/blob/master/Commands.md#git-effort>.
+
+- Display each file in the repository, showing commits and active days:
+
+`git effort`
+
+- Display files modified by specific number of commits or more, showing commits and active days:
+
+`git effort --above {{5}}`
+
+- Display files modified by a specific author, showing commits and active days:
+
+`git effort -- --author"{{username}}"`
+
+- Display files modified at a specific time/date, showing commits and active days:
+
+`git effort -- --since='{{last month}}'`
```

Comment:
Why the mixed quotes/missing `=`?
```suggestion
- Display files modified by a specific author, showing commits and active days:

`git effort -- --author="{{username}}"`

- Display files modified at a specific time/date, showing commits and active days:

`git effort -- --since="{{last month}}"`
```